File: KeyboardInput/BMICalculator/BMICalculator.java
import java.util.Scanner;

public class BMICalculator
{
  public static void main( String[] args )
  {
    Scanner keyboard = new Scanner(System.in);

    int FHeight, IHeight, PWeight;
    double MHeight, KWeight;

    System.out.print( "Your height (feet only): " );
    FHeight = keyboard.nextInt();

    System.out.print( "Your height (inches): " );
    IHeight = keyboard.nextInt();

    System.out.print( "Your weight in pounds: " );
    PWeight = keyboard.nextInt();

    System.out.println( "" );
    System.out.println( "Your BMI is " + ( PWeight * 0.453592 / ( ( FHeight * 0.3048 ) + ( IHeight * 0.0254 ) ) / ( ( FHeight * 0.3048 ) + ( IHeight * 0.0254 ) ) ) );
  }
}

File: C++/Conversion01/BMICalculator/BMICalculator.cpp
#include <iostream>
using namespace std;

int main() {
    int foot, inch, pound;

    cout << "Your height (feet only): ";
    cin >> foot;
    cout << "Your height (inches): ";
    cin >> inch;
    cout << "Your weight in pounds: ";
    cin >> pound;
    cout << "\n6
    Your BMI is " << ( pound * 0.4535 / ( foot * 0.3048 + inch * 0.0254 ) / ( foot * 0.3048 + inch * 0.0254 ) );
}